boilerplate/openshift/golang-osd-operator/configure-fips.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+MANAGER_DIR=${REPO_ROOT}/cmd/manager
+
+echo "Writing fips file at ${MANAGER_DIR}/fips.go"
+
+cat <<EOF>"${MANAGER_DIR}/fips.go"
+// +build fips_enabled
+
+// BOILERPLATE GENERATED -- DO NOT EDIT
+// Run 'make enable-fips' to regenerate
+
+package main
+
+import (
+  "fmt"
+  _ "crypto/tls/fipsonly"
+)
+
+func init() {
+  fmt.Println("***** Starting with FIPS crypto enabled *****")
+}
+EOF
+
+if ! grep -Fxq "ENV FIPS_ENABLED=true" build/Dockerfile
+then
+  echo "Writing to build/Dockerfile's to include FIPS_ENABLED=true"
+  sed -i '/^FROM quay.io\/app-sre\/boilerplate.*/a \\nENV FIPS_ENABLED=true' build/Dockerfile
```

Comment:
It puts `ENV FIPS_ENABLED=true` after the boilerplate builder `FROM` with a new line

Reply:
We have `if ! grep -Fxq "ENV FIPS_ENABLED=true" build/Dockerfile` on line 20 to check if it exist already

Reply:
D'oh, missed that, cool.